### DISCUSSION TITLE
visit site

### DISCUSSION THREAD

**Discussion by Manoj-das** [community/community#35406] [Other Feature Feedback, Questions, & Ideas]
i have accidently remove my visit site link how can i set back to it.

**Comment by DSDmark** [comment]
> i have accidently remove my visit site link how can i set back to it.

Can you specify more what you want?

If you remove it then check your history, you can find it

**Comment by DSDmark** [comment]
##  if you talking about your that repo :- 

![Screenshot from 2022-10-07 13-48-06](https://user-images.githubusercontent.com/85744816/194506967-98f3274b-9c87-4a78-a4c4-29f8e53a5205.png)

## then click on the environment section there is **github-page** click on it and new page open like this 

![Screenshot from 2022-10-07 13-52-27](https://user-images.githubusercontent.com/85744816/194507678-0160546e-15a9-4495-bec1-7a35acf38a9a.png)
 
> then click on **view deployment** button you can see your page

**Reply by radar-blip** [reply]
That isn't going to work, he turned index.html and style.css into folders. 
He needs to undo that.

**Reply by DSDmark** [reply]
Ya I know but i not sure what kind of link he is talking about I just guess

**Reply by radar-blip** [reply]
Ok well, I cant figure it right now. Have to go. Good luck!

**Comment by jfullstackdev** [comment]
your site ? this ? 

![image](https://user-images.githubusercontent.com/47092464/194709666-7f981e3b-9e18-4401-b67a-eaf52a0aeb4a.png)

https://manoj-das.github.io/Multi-PageRegistrationForm/index.html/
